### ISSUE TITLE
Typo in token URL

### ISSUE THREAD

**Issue by mcandre** [rust-lang/cargo#6225]
When I run `cargo publish` when logged out, I see a semi-helpful message:

```console
please visit https://crates.iome and paste the API Token below
```

The TLD has a typo: The correct domain is crates.io, not crates.iome. I think ya meant `crates.io/me`, no?

**Comment by ehuss** [comment]
Thanks, this should be fixed in 1.31 via #6078.

**Comment by alexcrichton** [comment]
Indeed!
